modules/riseBidAdapter.js

```diff
@@ -407,6 +407,7 @@ function generateGeneralParams(generalObject, bidderRequest) {
     device_type: getDeviceType(navigator.userAgent),
     ua: navigator.userAgent,
     is_wrapper: !!generalBidParams.isWrapper,
+    // TODO: fix auctionId leak: https://github.com/prebid/Prebid.js/issues/9781
     session_id: generalBidParams.sessionId || getBidIdParameter('auctionId', generalObject),
```

Comment:
since this sessionid isnt documented at https://docs.prebid.org/dev-docs/bidders/rise.html and can take any format; I think it is likely safe to fallback to bidderRequestId or getuuid here?